odra-casper/wasm-env/src/host_functions.rs

```diff
@@ -591,30 +723,22 @@ fn take_nth_caller_from_stack(n: usize) -> CallerInfo {
         .unwrap_or_revert()
 }
 
-fn create_constructor_group(contract_package_hash: ContractPackageHash) -> URef {
-    storage::create_contract_user_group(
-        contract_package_hash,
-        consts::CONSTRUCTOR_GROUP_NAME,
-        1,
-        Default::default()
-    )
-    .unwrap_or_revert()
-    .pop()
-    .unwrap_or_revert()
+fn create_user_group(contract_package_hash: ContractPackageHash, group_label: &str) -> URef {
```

Comment:
Rename to `create_contract_user_group`. Not it is the same as one of variables in code.

Reply:
Ok